src/localizations/no_nb.json

```diff
@@ -137,7 +137,7 @@
     "chosen_services": "Valgte tjenester",
     "alert_partially_delegable_header": "Du får ikke delegert hele tjenesten",
     "you_cant_delegate_these_rights": "Disse rettighetene kan du ikke delegere:",
-    "action_bar_adjust_rights_text": "Hvis du vil begrense tilgangen til en rettighet kan du klikke på sjekkmerkene nedenfor for å fjerne dem",
+    "action_bar_adjust_rights_text": "Hvis du vil begrense tilgangen til en rettighet kan du klikke på knappene nedenfor for å fjerne dem",
```

Comment:
Didn't we talk about dropping this line here and baking it into the header below? Or did this change?
Picture from the design sync:
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/f287e729-0a38-481b-ab0e-939cf831bc7f)

Reply:
This is the last updated design:

<img width="902" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/9d1bb497-5fe5-4f3c-a860-d294ee7bff90">